microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -93,22 +61,92 @@ def verify_ltp_lite(
                     node.tools[Lsblk].find_disk_by_mountpoint(mountpoint).device_name
                 )
 
-        # run ltp lite tests
+        if not test_list:
+            # empty test_list means full test, require long timeout
+            run_timeout = 72000
```

Comment:
It won't work, because the test case metadata has another timeout. This value couldn't be longer than that. 72000 means  20 hours, it's too long to finish all cases.